tests/test_validate.py

```diff
@@ -17,17 +17,16 @@ def test_validator_simple():
     schema = {
         "type": "object",
         "properties": {
-            "price": {"type": "number"},
             "name": {"type": "string"},
         },
-        "required": ["price", "name"]
+        "required": ["name"]
```

Comment:
dtCost, name

Reply:
plecos only does metadata section, and not the services section.  there is no reason to have a dtCost for metadata